data_science/scripts/toggle_asg.py

```diff
@@ -3,63 +3,145 @@
 """
 Start/stop all the instances in an autoscaling group.
 
-Usage: toggle_asg.py (--start | --stop | --status) [--type=(p2 | t2)]
+Usage: toggle_asg.py (--start | --stop | --enable-overnight | --disable-overnight) [--type=(p2 | t2)]
+       toggle_asg.py [--type=(p2 | t2)]
+
 
 Actions:
   --start                 Start the autoscaling group (set the desired count to 1).
   --stop                  Stop the autoscaling group (set the desired count to 0).
-  --status                Report the current side of autoscaling group.
+  --enable-overnight      Keeps the autoscaling group desired count at 1 (disables auto scale down)
```

Comment:
Describe in terms of user visible effects: “Keeps the ASG running overnight (disables auto scale down)” and “Stops the ASG from running overnight (enables auto scale down)”